tests/unit/test_utils.py

```diff
@@ -24,3 +42,21 @@ def test_get_future(self):
 
         assert len(responses) == 5
         assert sorted(responses)[0][1] == "ZeroDivisionError('division by zero')"
+
+
+class TestProgressBar:
+
+    @pytest.mark.usefixtures("hide_available_pkg")
+    def test_if_no_package(self):
+        with pytest.raises(ModuleNotFoundError):
+            _create_progress_bar("Test", 10)
+
+    def test_create_progress_bar(self):
+        pytest.importorskip("rich")
+
+        progress, task_id = _create_progress_bar("Test", 10)
+        print(progress, task_id)
+
+        assert len(progress.columns) == 3
+        assert task_id == 0
```

Comment:
Is the `task_id` a deterministic generator? Do we know that this will always be `0`?

Reply:
The `tasks` object is a list and the ID is just the position on it. Since it is created just one task it will be always 0.

Reply:
Thanks :+1: